atlas/consensus/quorum.go

```
@@ -0,0 +1,330 @@
+			Name:  "self",
+			Value: atlas.CurrentOptions.ServerId,
+		})
+		if err != nil {
+			return nil, err
+		}
+
+		nodes := make([]*QuorumNode, len(results.Rows))
+		for i, row := range results.Rows {
+			var client ConsensusClient
+			var closer func()
+			client, err, closer = getNewClient(row.GetColumn("address").GetString() + ":" + row.GetColumn("port").GetString())
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)